### PR TITLE
ENH: added faq page

### LINKED ISSUE #18: Add FAQ Page
Most of the details are pretty much same as last year [FAQ](https://in.pycon.org/2023/faq/).

### PR DESCRIPTION
**Pull request by Schefflera-Arboricola**
closes https://github.com/pythonindia/inpycon2024/issues/18

used 2023's faq.md as reference, LMK if I missed any updates. 
Could you please help me update some of the links, as I don't know which ones are old and which ones don't exist? 

Thank you :)

### CODE REVIEW

File: src/_pages-content/faq.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+#### Is there a different ticket for the Workshops?
+Yes. A separate ticket has to be purchased for each workshop. The following is included as part of the workshop ticket:
+- Access to the specific workshop
+- Lunch
+- High-Tea and Snacks
+- Access to Kid's engagement (by Volunteers) at the Venue
+
+#### Is there a different ticket for the Devsprints?
+No, you do not need a ticket for Devsprints. Devsprints are open to all participants and is free to attend.
```
Comment (anistark) [inline src/_pages-content/faq.md]: We don't know this yet. Better to skip this for now or write TBD something.

File: src/_pages-content/faq.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+#### Is accommodation included in the ticket?
+No, it doesn’t include accommodation.
+
+#### Would there be any spot registration at the venue?
+No, there won’t be any spot registrations available at the conference venue.
```
Comment (anistark) [inline src/_pages-content/faq.md]: Yes there might be spot registrations as discussed on last call. Please refer to meeting notes.

File: src/_pages-content/faq.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+2. **Poster session:** Poster session is a 1-hour session during which presenters use a graphic poster and a laptop to present their topic.
+3. **Open spaces:** Open spaces are where people have informal presentations and discussions alongside the talk tracks. Open spaces slots can be booked at the venue, on the day of the event.
+4. **Lightning talks:** Lightning talks is a 1-hour session. Lightning talks are rapid 5 min presentations, presented to the entire conference crowd, a great way to get a message across to the entire audience.
+5. **Sponsor stalls:** Sponsor stalls are where you can meet companies supporting the conference. These companies use Python and are looking for Python developers. You can visit their stalls to get to know more about them, and the openings in their organization.
```
Comment (anistark) [inline src/_pages-content/faq.md]: Do we call them `stalls` or `booths` ?

**Review by anistark — CHANGES_REQUESTED** [review]

**Comment by Schefflera-Arboricola** [comment]
@anistark thanks for the reviews. 
I've updated all the links as well and I think it's ready now. Please let me know if anything else needs to be added/edited.

> Do we call them `stalls` or `booths` ?

idk, Please let me know.

Thank you :)

File: src/_pages-content/faq.md (hunk 3 above)
Comment (bhansa) [reply src/_pages-content/faq.md]: booth is better.

File: src/_pages-content/faq.md [hunk 4]
```diff
@@ -32,10 +32,10 @@ All announcements related to the conference are posted to [PyCon India Announce
 The call for Proposals is now open. Please visit the [CFP page](https://in.pycon.org/cfp/2024/proposals/) to submit your talk/workshop/ proposal.
 
 #### I am a first-time speaker & need some help in getting started. What should I do?
-Check out the [Guide to Speaking](https://in.pycon.org/blog/2023/guide-to-speaking.html) to understand more about the process. For any doubts, feel free to reach out to us!
+Check out the [Speaker best practices](https://github.com/pythonindia/junction/wiki/Speaker-best-practices) and the [PyCon 2023's Guide to Speaking](https://in.pycon.org/blog/2023/guide-to-speaking.html) to understand more about the process. For any doubts, feel free to reach out to us!
```
Comment (anistark) [inline src/_pages-content/faq.md]: Add only 1 link. This should ideally by on navigator but for now junction link is enough. If there's something missing in that content-wise, that can be added from blog link.

File: src/_pages-content/faq.md [hunk 5]
```diff
@@ -59,13 +59,13 @@ Yes. A separate ticket has to be purchased for each workshop. The following is i
...
 #### Is there a different ticket for the Devsprints?
-No, you do not need a ticket for Devsprints. Devsprints are open to all participants and is free to attend.
+TBD
 
 #### Is accommodation included in the ticket?
 No, it doesn’t include accommodation.
 
 #### Would there be any spot registration at the venue?
-No, there won’t be any spot registrations available at the conference venue.
+Yes, there might be some limited number of spot registrations available at the conference venue.
```
Comment (anistark) [inline src/_pages-content/faq.md]: This is decided now. There's will be a few limited number of on-spot registrations on first-come-first basis. But the goodies are subject to availability.

**Review by anistark — CHANGES_REQUESTED** [review]

**Review by anistark — APPROVED** [review]

**Comment by Schefflera-Arboricola** [comment]
-just a gentle reminder

Thx!

**Pull request merged**